app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -2043,6 +2043,8 @@ public void run() {
      */
     private void openLogin() {
         Intent i = new Intent(this, LoginActivity.class);
+        String server_url = getSharedPreferences(LoginActivity.Prefs_name,MODE_PRIVATE).getString(LoginActivity.Prefs_server,null);
```

Comment:
Using this always returning the default instead of the current url. I made sure that I pull out values before they flushed out still getting.